webapp/javascript/ui/Sidebar.scss

```diff
@@ -130,3 +130,11 @@ $sidebar-collapsed-width: 54px !default;
   .pro-inner-item {
   padding-left: 42px;
 }
+
+.pro-sidebar > .pro-sidebar-inner > .pro-sidebar-layout .pro-sidebar-header {
+  border-bottom: 1px solid var(--ps-ui-element-bg-highlight);
+}
+
+.pro-sidebar > .pro-sidebar-inner > .pro-sidebar-layout .pro-sidebar-footer {
+  border-top: 1px solid var(--ps-ui-element-bg-highlight);
+}
```

Comment:
Is it possible to simplify such a classname chain? :)

`.pro-sidebar > .pro-sidebar-inner > .pro-sidebar-layout .pro-sidebar-footer`
...

> `rgba(var(--ps-ui-element-bg-highlight), 0.2)`

 they don't work with each other because `var()` is CSS variable, but `rgba()` is SASS modifier (works only with SASS variables ($color)),

I'd transform this color to `#RGBA` and put it to variables.css, @eh-am right ?

Reply:
can't simplify it because selector needs to has same strength

Reply:
sorry whats the issue here ? you said the reason why "it" happens and not sure what "it" is referring to

Reply:
@Rperry2174 the bug with sepatarion lines

Reply:
we basically have styles for the border, but they are not reflected in the UI